files/SQLite.props

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup Condition="!Exists('$(MSBuildProjectName).project.json') And !Exists('project.json') And '$(Platform)' == 'AnyCPU'">
+  <ItemGroup Condition="!Exists('$(MSBuildProjectName).project.json') And !Exists('project.json') And '$(Platform)' == 'AnyCPU' And '$(ExcludeSqliteNative)' != 'true' ">
```

Comment:
This seems silly. Why install the package if you don't want it? (I know why, but from the perspective of just this package it seems like the wrong thing to enable.)